.circleci/config.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
       # Then, build the actual app in a container shared between Linux and Windows
       - run: docker pull livepeerci/build:latest || echo 'no pre-existing cache found'
-      - run: docker build -t livepeerci/build:latest --cache-from=livepeerci/build:latest -f docker/Dockerfile.build .
+      - run: docker build --build-arg HIGHEST_CHAIN_TAG=rinkeby -t livepeerci/build:latest --cache-from=livepeerci/build:latest -f docker/Dockerfile.build .
```

Comment:
I'd advise against it.  The way it works as-is ensures that the env var is defined both during the build and in the running release image.  It will be the only way to tell after the fact what variable was set during the build.

Reply:
OOPS...  this is a typo - will fix it now

Reply:
Couldn't we tell what variable was set by looking at the [listed env vars](https://github.com/livepeer/go-livepeer/blob/master/.circleci/config.yml#L22)? Seems like the only difference in these two approaches is setting `HIGHEST_CHAIN_TAG` in the `environment` section vs. setting `HIGHEST_CHAIN_TAG` directly when passing the `--build-arg` flag.

Do you prefer setting the value for `HIGHEST_CHAIN_TAG` closer to where it is used as opposed to in a separate `environment` section? Note: We already set `HIGHEST_CHAIN_TAG` in a separate `environment` section when [creating the binary](https://github.com/livepeer/go-livepeer/blob/master/.circleci/config.yml#L63).

Reply:
On second thought,
```
docker build --build-arg HIGHEST_CHAIN_TAG -t livepeerci/build:latest --cache-from=livepeerci/build:latest -f docker/Dockerfile.build .
```
...might be okay - I will push up the change.

Reply:
done in 003fb82a053d9652243aba0dea69b439234bfb5b - it should be fine.  Thanks for the suggestion @yondonfu :-)

Reply:
our paths crossed here - Yes, I prefer setting the env var closer to where it is actually used, but in this case it makes sense to use the CircleCI environment,